src/tests/JIT/opt/OptimizeBools/optboolsreturn.csproj

```
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestPriority>1</CLRTestPriority>
```

I would make this new test Pri0 for now (the default) so it runs in normal CI.
```suggestion
    <CLRTestPriority>1</CLRTestPriority>
```

---

Done.